modules/tinymce/src/themes/silver/main/ts/ui/menus/menubar/Integration.ts

```diff
@@ -25,7 +25,7 @@ const defaultMenus: Record<string, MenuSpec> = {
   view: { title: 'View', items: 'code | visualaid visualchars visualblocks | spellchecker | preview fullscreen | showcomments' },
   insert: { title: 'Insert', items: 'image link media addcomment pageembed template codesample inserttable | charmap emoticons hr | pagebreak nonbreaking anchor tableofcontents footnotes | mergetags | insertdatetime' },
   format: { title: 'Format', items: 'bold italic underline strikethrough superscript subscript codeformat | styles blocks fontfamily fontsize align lineheight | forecolor backcolor | language | removeformat' },
-  tools: { title: 'Tools', items: 'spellchecker spellcheckerlanguage | autocorrect capitalization | a11ycheck code wordcount' },
+  tools: { title: 'Tools', items: 'spellchecker spellcheckerlanguage | autocorrect capitalization | a11ycheck code wordcount typography' },
```

Comment:
Is that really where the design team wants it? I would've thought up near spell checking, or in it's own section at the end rather than tacked on after word count was more appropriate.

- [ ] Check item location with design and/or product